agent/internal/docker.go

```diff
@@ -202,6 +208,48 @@ func (d *dockerActor) pullImage(ctx *actor.Context, msg pullImage) {
 	ctx.Tell(ctx.Sender(), imagePulled{})
 }
 
+func (d *dockerActor) getDockerAuths(
+	ctx *actor.Context,
+	expconfReg *types.AuthConfig,
+	image reference.Named,
+) (types.AuthConfig, error) {
+	imageDomain := reference.Domain(image)
+	// Try expconf registry auth config.
+	if expconfReg != nil {
+		didNotPassServerAddress := expconfReg.ServerAddress == ""
+		if didNotPassServerAddress {
+			d.sendAuxLog(ctx, "warning setting registry_auth without registry_auth.serveraddress "+
```

Comment:
it'd be great to start passing "level" parameter into `sendAuxLog`, `aproto.ContainerLog` struct and then `sproto.ContainerLog.Level` on master side so this warning becomes a warning from log filtering perspective as well.

Reply:
I agree though I think I'm going to do it in a follow up PR since this one is already somewhat large